### PR TITLE
fix: temp remove profile create on base

### LINKED ISSUE #10401: Temp fix: hide create profile on Base
### Is your feature request related to a problem?

as @exezbcz said in - #10382 that we are waiting for evm connector to be merged
I do not want to have non-working feature on the KodaDot. 

### Describe the solution you would like

Temporary hide create profile on base :) 

### Describe alternatives you have considered

Do not merge EVM connector :)

### Screenshots

_No response_

### Are there opened related issues?

_No response_

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [Koda - Generative Art Marketplace](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #10401

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

**Review by vikiival — CHANGES_REQUESTED** [review]
`consola warn ('[ProfileDetail.vue] Profiles on base are not enabled yet')`

**Comment by vikiival** [comment]
Missing scr

**Comment by Jarsen136** [comment]
> Missing scr

Screenshot added

**Review by vikiival — APPROVED** [review]

File: components/profile/ProfileDetail.vue [hunk 1]
```diff
@@ -483,7 +484,7 @@ const followConfig: ButtonConfig = {
       initiatorAddress: accountId.value,
       targetAddress: id.value as string,
     }).catch(() => {
-      if (isBase.value) {
+      if (isSub.value) {
```
Comment (vikiival) [inline components/profile/ProfileDetail.vue]:
```suggestion
      if (isEvm.value) {
```

**Review by vikiival — CHANGES_REQUESTED** [review]
Incorrect check

File: components/profile/ProfileDetail.vue (hunk 1 above)
Comment (Jarsen136) [reply components/profile/ProfileDetail.vue]: It's the old commit on the changes. Could you reload this page?

**Review by vikiival — APPROVED** [review]

File: components/profile/ProfileDetail.vue (hunk 1 above)
Comment (vikiival) [reply components/profile/ProfileDetail.vue]: Yup I had old changes

**Pull request merged**